app/services/phone_confirmation/confirmation_session.rb

```diff
@@ -28,7 +28,7 @@ def regenerate_otp
     end
 
     def matches_code?(candidate_code)
-      Devise.secure_compare(candidate_code, code)
+      Base32::Crockford.normalize(candidate_code) == Base32::Crockford.normalize(code)
```

Comment:
should we keep the secure compare just in case?

```suggestion
      Devise.secure_compare(Base32::Crockford.normalize(candidate_code), Base32::Crockford.normalize(code))
```

Reply:
Rubocop was mad about line length, so added in 41565ce